awx/ui/src/screens/Job/JobOutput/useJobEvents.js

```diff
@@ -25,7 +25,12 @@ export const REBUILD_TREE = 'REBUILD_TREE';
 export const TOGGLE_COLLAPSE_ALL = 'TOGGLE_COLLAPSE_ALL';
 export const SET_CHILDREN_SUMMARY = 'SET_CHILDREN_SUMMARY';
 
-export default function useJobEvents(callbacks, jobId, isFlatMode) {
+export default function useJobEvents(
+  callbacks,
+  jobId,
+  hasChildrenSummary,
```

Comment:
This addresses a failed API request for `children_summary` on job types that do not have that end point.  @keithjgrant

Reply:
`hasChildSummary` shouldn't be necessary, since it will always be in flat mode for job types that don't have child summaries